src/Lifecycle/index.js

```diff
@@ -23,8 +23,8 @@ import isProbablyLightningComponent from '../helpers/isProbablyLightningComponen
 import { initBaseEvents } from './base'
 import { initLightningEvents } from './lightning'
 
-const supportedStates = ['init', 'active', 'pause', 'background', 'close']
-let currentState = 'init'
+const supportedStates = ['loading', 'active', 'paused', 'background', 'closed']
```

Comment:
closed is a final state, the app will never know its closed. It can only have the intent of closing or an external request to close.

Reply:
good point.